tests/base.py

```diff
@@ -179,6 +179,14 @@ def get_admin_location_id(self):
         location = Location.query.filter_by(name=user.location).first()
         return location.id
 
+    def assign_resource(self, query):
```

Comment:
@kwanj-k ,I suppose we don't need this method since there is a method that works the same in CommonTestCases class.

Reply:
@TeamoreA, this is the method that was discussed during the team sync for assigning a resource to a room so that updating or querying can be tested

Reply:
@TeamoreA as @d-kahara has mentioned, this helper method is solely meant to just assign a resource to a room during testing, something that is not present at the moment

Reply:
My idea is that you can reuse the existing methods like `admin_token_assert_equal` @d-kahara .

Reply:
Thanks for the feedback, this has been worked on @TeamoreA